src/__tests__/funders.int.test.ts

```diff
@@ -20,14 +21,8 @@ describe('/funders', () => {
 			await agent.get('/funders').expect(401);
 		});
 
-		it('returns an empty bundle when no data is present', async () => {
-			await agent.get('/funders').set(authHeader).expect(200, {
-				entries: [],
-				total: 0,
-			});
-		});
-
```

Comment:
Did this need to be removed?

Reply:
Yes indeed -- there is no longer a case of there being no data now that we have a system funder.